Core/include/Acts/Surfaces/ConvexPolygonBounds.hpp

```diff
@@ -112,13 +112,13 @@ class ConvexPolygonBounds : public ConvexPolygonBoundsBase {
 
   /// Return the vertices
   ///
-  /// @param lseg the number of segments used to approximate
+  /// @param ignoredSegments the number of segments used to approximate
   /// and eventually curved line
   ///
   /// @note the number of segments is ignored in this representation
   ///
   /// @return vector for vertices in 2D
-  std::vector<Vector2> vertices(unsigned int lseg = 1) const final;
+  std::vector<Vector2> vertices(unsigned int ignoredSegments = 0u) const final;
```

Comment:
Interface wise this integer has a different meaning depending on the surface right? I wonder if that might be surprising if you call this method from the base class. Potentially all the different params could be bundled up in a struct and passed as options?

Reply:
For surfaces without curves it's actually ignored - which is why it is renamed.